### PR TITLE
Nc games 4

### PR DESCRIPTION
**Pull request by MaxD-P**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -36,4 +36,36 @@ describe('GET /api', () => {
...
     })
+})
+
+describe('GET /api/reviews/:review_id', () => {
+    test('GET - STATUS: 200 - responds with review object of corresponding review_id endpoint', () => {
+        return request(app).get('/api/reviews/1')
+        .expect(200)
+        .then((res) => {
+            expect(typeof res.body).toBe('object');
+            expect(res.body.review.title).toBe('Agricola');
```
Comment (Beth-McCormack) [inline __tests__/app.test.js]: An expect statement for the article_id of this object is also important here 🙂

File: __tests__/app.test.js [hunk 2]
```diff
@@ -36,4 +36,36 @@ describe('GET /api', () => {
...
+            expect(res.body.review.designer).toBe('Uwe Rosenberg');
+            expect(res.body.review.owner).toBe('mallionaire');
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:49]: to allow for later extension of the endpoint, this test can use a flexible matcher eg expect.objectContaining or .toMatchObject

File: __tests__/app.test.js [hunk 3]
```diff
@@ -36,4 +36,36 @@ describe('GET /api', () => {
             expect(res.body).toEqual(endpoints)
         })
     })
+})
+
+describe('GET /api/reviews/:review_id', () => {
+    test('GET - STATUS: 200 - responds with review object of corresponding review_id endpoint', () => {
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:42]: to allow for later extension of the endpoint, this test should use a flexible matcher eg expect.objectContaining or .toMatchObject

**Review by Beth-McCormack — CHANGES_REQUESTED** [review]
Nice work! Just a couple of comments for you, please submit another PR once these are done 🙂

**Review by leekli — APPROVED** [review]
Approved

**Pull request merged**